.editorconfig

```diff
@@ -14,3 +14,7 @@ trim_trailing_whitespace = false
 [*.yml]
 indent_style = space
 indent_size = 2
+
+[*.rb]
+indent_style = space
```

Comment:
The `indent_style = space` is already defined by `[*]` so there is no need to declare it again.

But given that this was previously rejected, why don't you keep this PR more scoped by just fixing #910 and sending this in an independent PR to help with the discussion?